Update welcome email

ThatSnowman opened the pull request:
changed the registration email to the new mail provided by the new board

ThatSnowman committed:
new_registration_email.txt

ThatSnowman committed:
Update registration_email.txt

LenaWil commented on membership_file/templates/membership_file/registration/registration_email.txt:
```diff
@@ -1,17 +1,51 @@
 If this email was not meant for you, then do let us know.
 
 See you soon!
-
-{{ sender.name }}
-{{ sender.description }}{% if sender.extra_description %}
-{{ sender.extra_description }}{%endif%}
-ESRG Knights of the Kitchen Table
-{% endautoescape %}
```
Why is this removed?

ThatSnowman replied:
it shouldn't but I will see I can fix it again

LenaWil reviewed (changes requested):
The test is breaking, and I think that might be because you removed an `{%endif%}` & `{% endautoescape %}`.

ThatSnowman committed:
update_fix_registration_email.txt

LenaWil commented on membership_file/templates/membership_file/registration/registration_email.txt:
```diff
@@ -1,17 +1,51 @@
+**Activities**
+ The knights hold some weekly and monthly activities:
+
+**•	Weekly Board Game Evenings –** Every evening at 19:30, you can join us in the living room to play board games. We recommend showing up at least 5 to 10 minutes early to find out about upcoming activities, socialise and find a group to play with.
+
+**•	Wednesday activities –** Every Wednesday, at 19:30 we (try to) organise a special activity,
+o	This includes role-playing one-shots every first Wednesday of the month.
+o	Various trading card activities
+o	Role-playing workshops
+o	Special one-off activities
+
+**•	Sword Fighting Training –** Every two weeks, we host a training session where you can practise sword fighting — or, more accurately, hit each other with foam swords.
+o	This activity starts at 19:20 and once again, we recommend showing up a bit earlier. 
+
+**•	Messing Schaar:** Arts & Crafts – Once a month, members gather for an arts & crafts day, where you can get help finding the right materials to buy, and work on cutting and sewing them into your LARP costume. You can also work on other fun crafting projects!
+
+**•	**Members weekend –** Twice a year, we go away for a themed weekend of Knights fun. During the weekend we organise a LARP, multiple role playing one-shots, other unique activities, and plenty of time to play your favourite board games!
+
+**•	Special Events –** We also run special events throughout the year, such as:
+o	**Find your Roleplaying group:** An event where you can find a group to play a role-playing campaign like DND with.
+	If you would like to run a campaign and pitch it at this event, please contact [Insert contact info]
+
+o	**Crazy cloak construction:** A full day dedicated to creating your own cloak, with no prior experience required.
+o	**Mini-painting workshop**
+
+**Communication Channels**
+
+**•	Discord –** Our main hub for chatting, finding games, and staying up-to-date.
+  [https://discord.gg/rcnjVzZsBc]
+ Once you join, contact a board member to get access to all Knights channel
+
+**•	Squire – Our Web App –** We use Squire to organise games, track achievements, manage membership, and sign up for events.
```
Also there is a lot of formatting like this: `**•`. Are you sure that’s correct?

ThatSnowman replied:
it should make the text in between the ** go bold, so yes that is correct

LenaWil replied:
Okay please don’t take this personal Erik, but @EricTRL you can double check the formatting?

(if not, it’s fine, I can try to setup a squire instance+mail server to try it locally)

LenaWil replied:
Okay I changed it for the markdown to be consistent (the above text now refers to something outdated), but the formatting seems to be plaintext, at least looking at the original.

Maybe html would be accepted though?

LenaWil replied:
throwing markdown into a html converter would let to something like this:
```html
<p>{% load i18n %}{% autoescape off %} <strong>Knights of the Kitchen
Table – Welcome Letter</strong> Hail {{ member.first_name }}! We’re
excited to have you join our community of tabletop, role-playing, and
board game enthusiasts! This email contains all the essential info to
get you started.</p>
<h2 id="activities">Activities</h2>
<p>The knights hold some weekly and monthly activities:</p>
<ul>
<li><p><strong>Weekly Board Game Evenings –</strong> Every evening at
19:30, you can join us in the living room to play board games. We
recommend showing up at least 5 to 10 minutes early to find out about
upcoming activities, socialise and find a group to play with.</p></li>
<li><p><strong>Wednesday activities -</strong> Every Wednesday, at 19:30
we (try to) organise a special activity,</p>
<ul>
<li>This includes role-playing one-shots every first Wednesday of the
month.</li>
<li>Various trading card activities</li>
<li>Role-playing workshops</li>
<li>Special one-off activities</li>
</ul></li>
<li><p><strong>Sword Fighting Training –</strong> Every two weeks, we
host a training session where you can practise sword fighting — or, more
accurately, hit each other with foam swords.</p>
<ul>
<li>This activity starts at 19:20 and once again, we recommend showing
up a bit earlier.</li>
</ul></li>
<li><p><strong>Messing Schaar:</strong> Arts &amp; Crafts – Once a
month, members gather for an arts &amp; crafts day, where you can get
help finding the right materials to buy, and work on cutting and sewing
them into your LARP costume. You can also work on other fun crafting
projects!</p></li>
<li><p><strong>Members weekend –</strong> Twice a year, we go away for a
themed weekend of Knights fun. During the weekend we organise a LARP,
multiple role playing one-shots, other unique activities, and plenty of
time to play your favourite board games!</p></li>
<li><p><strong>Special Events –</strong> We also run special events
throughout the year, such as:</p>
<ul>
<li><p><strong>Find your Roleplaying group:</strong> An event where you
can find a group to play a role-playing campaign like DND with. If you
would like to run a campaign and pitch it at this event, please contact
[Insert contact info]</p></li>
<li><p><strong>Crazy cloak construction:</strong> A full day dedicated
to creating your own cloak, with no prior experience required.</p></li>
<li><p><strong>Mini-painting workshop</strong></p></li>
</ul></li>
</ul>
<p><strong>Communication Channels</strong></p>
<ul>
<li><p><strong>Discord –</strong> Our main hub for chatting, finding
games, and staying up-to-date.<br />
<a href="https://discord.gg/rcnjVzZsBc">Invite link here</a><br />
Once you join, contact a board member to get access to all Knights
channels.</p></li>
<li><p><strong>Squire – Our Web App –</strong> We use Squire to organise
games, track achievements, manage membership, and sign up for
events.<br />
You can register an account for {{ member.get_full_name }} through the
following link. Doing so will automatically link it to your membership
data.</p></li>
</ul>
<p>{{ protocol }}://{{ domain }}{% url ‘membership:link_account/confirm’
uidb64=uid token=token %}</p>
<p>You can browse upcoming events, access the full list of our board
games, and sign up for certain activities.</p>
<ul>
<li><strong>WhatsApp / Telegram Announcement Channel –</strong> Join one
of our announcement channels to make sure you never miss an event!<br />
<a
href="https://chat.whatsapp.com/G47fmdu3Ufh5rpBI9XQY7v?mode=ems_copy_c">WhatsApp</a><br />
<a href="https://t.me/KightsAankondigingen">Telegram</a></li>
</ul>
<p>If this email was not meant for you, then do let us know.</p>
<p>See you soon!</p>
<p>{{ sender.name }} {{ sender.description }}{% if
sender.extra_description %} {{ sender.extra_description }}{%endif%} ESRG
Knights of the Kitchen Table {% endautoescape %}</p>
```

LenaWil committed:
Make the formatting in registration_email.txt more markdown like

LenaWil commented on membership_file/templates/membership_file/registration/registration_email.txt:
```diff
@@ -1,11 +1,51 @@
+
+- **Special Events –** We also run special events throughout the year, such as:
+  - **Find your Roleplaying group:** An event where you can find a group to play a role-playing campaign like DND with.
+ 	If you would like to run a campaign and pitch it at this event, please contact [Insert contact info]
```
todo replace insert contact info

ThatSnowman committed:
Update dynamic_preferences_registry.py

the new global preferences for the welcome email

ThatSnowman commented:
ToDo remove hardcode membership fee amount, replace it with global preference

EricTRL committed:
Update global URL preferences

Renamed some things

EricTRL committed:
Configure HTML variant registration email

Allow registration email to be sent as HTML.
Modified markdown-format back into plaintext.

Extended password validation.

EricTRL committed:
Merge branch 'master' into Erik-Email-Patch
